qa/L0_backend_python/io/requested_output_model/model.py

```diff
@@ -0,0 +1,122 @@
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+#
+# This test case was added based on a prior crash. DO NOT MODIFY!
+#
+
+import json
+import traceback
+
+import numpy as np
+import triton_python_backend_utils as pb_utils
+
+
+def get_valid_param_value(param, default_value=""):
+    value = param.get("string_value", "")
+    return default_value if value.startswith("${") or value == "" else value
+
+
+class TritonPythonModel:
+    def initialize(self, args):
+        # Parse model configs
+        model_config = json.loads(args["model_config"])
+
+        self.output_config = pb_utils.get_output_config_by_name(
+            model_config, "text_output"
+        )
+
+        self.output_dtype = pb_utils.triton_string_to_numpy(
+            self.output_config["data_type"]
+        )
+
+        self.decoupled = pb_utils.using_decoupled_model_transaction_policy(model_config)
+
+        self.logger = pb_utils.Logger
+
+    def create_triton_tensors(self, index):
+        x = "bla" + str(index)
+        output = [x.encode("utf8")]
+        np_output = np.array(output).astype(self.output_dtype)
+        seq_idx = np.array([[0]]).astype(np.int32)
+
+        t1 = pb_utils.Tensor("text_output", np_output)
+        t2 = pb_utils.Tensor("sequence_index", seq_idx)
+        tensors = [t1, t2]
+        return tensors
+
+    def create_triton_response(self, index):
+        tensors = self.create_triton_tensors(index)
+        return pb_utils.InferenceResponse(output_tensors=tensors)
+
+    def execute(self, requests):
+        responses = []
+        for request in requests:
+            if self.decoupled:
+                response_sender = request.get_response_sender()
+            try:
+                for index in range(0, 1):
+                    triton_response = self.create_triton_response(index)
+                    # output_tensors = triton_response.output_tensors()
+                    if self.decoupled:
+                        response_sender.send(triton_response)
+                    else:
+                        responses.append(triton_response)
+
+                if self.decoupled:
+                    response_sender.send(
+                        flags=pb_utils.TRITONSERVER_RESPONSE_COMPLETE_FINAL
+                    )
+
+            except Exception:
+                # self.logger.log_info(f"error response")
```

Comment:
Do we need to keep this?

Reply:
No, it is cleaned up: [Clean up on the test model](https://github.com/triton-inference-server/server/pull/7866/commits/d5f668cb7d814576d5e16dc982770dda1d5bc5fb)